tyr/clusters/autoscaling.py

```diff
@@ -10,6 +10,7 @@ class AutoScaler(object):
     '''
     def __init__(self, launch_configuration,
                  autoscaling_group,
+                 instance_type,
```

Comment:
I don't think this line is actually needed - it seems the node_obj passes this setting anyway - the problem was just in the launch configuration constructor.